hw/ip/hmac/doc/_index.md

```diff
@@ -226,9 +230,8 @@ software under `sw/`.
 
 This section of the code describes initializing the HMAC-SHA256, setting up the
 interrupts, endianess, and HMAC, SHA-256 mode. {{< regref "CFG.endian_swap" >}} reverses
-byte-oder of input message when the software writes message into the FIFO.
-{{< regref "CFG.digest_swap" >}} is to reverse the result of the HMAC or SHA hash. It doesn't
-reverse the byte-order of the internal logic but to the registers only.
+the byte-order of input words when software writes into the message FIFO.
+{{< regref "CFG.digest_swap" >}} reverses the byte-order in the final HMAC or SHA hash.
```

Comment:
I don't recall if it's covered elsewhere in this documentation, but it might be good to note here that the defaults for message/digest endianness are flipped from each other, so if you want both little-endian OR both big-endian you need to set one but not the other of these swap bits, which is a little counter-intuitive (to me at least). Relevant code from the dif is here:
https://github.com/lowRISC/opentitan/blob/bb1701c2872d1ee23e0b41a3e27d0a3c08174559/sw/device/lib/dif/dif_hmac.c#L60-L84